### ISSUE TITLE
nginx: [emerg] unknown "connection_upgrade" variable

### ISSUE THREAD

**Issue by systemx-xx** [Fmstrat/plex-ssl#11]
In Ubuntu 14.04.02 LTS, Nginx 1.4.6

Running configuration test is giving me,

nginx: [emerg] unknown "connection_upgrade" variable

Ive tried to set the connection_upgrade to https, while this does work I can access my server on port 33443 over http without https perfectly, is this correct behaviour ?

**Comment by JohnKiel** [comment]
Yes.  Unfortunatley, because of the way plex works, some things still require http.  (iOS clients, roku clients, pings from plex.tv, etc.)

The readme documents some of the issues we ran into.
